test/lib/operation.go

```diff
@@ -121,6 +122,38 @@ func (client *Client) sendFakeEventWithTracingToAddress(
 	return nil
 }
 
+// ConfigMapExists will check if a configmap exists in a specific namespace
+func (client *Client) ConfigMapExist(namespace string, names ...string) error {
```

Comment:
I think `validation.go` is a better place to hold these two functions.
Also I would suggest to change the function names to `CheckConfigMapsExist` and `CheckConfigMapsEqual`.